src/components/Discussion/Internal/Comment/Actions.js

```diff
@@ -137,6 +171,18 @@ export const Actions = ({ t, comment, onReply, onEdit }) => {
 
   return (
     <div {...styles.root}>
+      {onExpand && (
+        <IconButton
+          type='left'
+          onClick={onExpand}
+          title={t('styleguide/CommentActions/expand')}
+        >
+          <CommentCountIcon
+            count={comment.comments && comment.comments.totalCount}
+            small={true}
+          />
+        </IconButton>
+      )}
       {onReply && (
```

Comment:
sorry wrong location before, here is the right one:
```suggestion
      {onReply && !!displayAuthor && (
```
(for a long standing issue that guests see the reply icon)